internal/db/schema/migrations/oss/postgres/60/01_ldap.up.sql

```diff
@@ -309,6 +311,36 @@ comment on table auth_ldap_bind_credential is
 'auth_ldap_bind_credential entries allow Boundary to bind (aka authenticate) using '
 'the provided credentials when searching for the user entry used to authenticate.';
 
+-- auth_ldap_account_attribute_map entries are the optional attribute maps from custom
+-- attributes to the standard attribute of fullname and email.  There can be 0 or more
+-- for each parent ldap auth method. 
+create table auth_ldap_account_attribute_map (
+  create_time wt_timestamp,
+  ldap_method_id wt_public_id 
+    constraint auth_ldap_method_fkey
+    references auth_ldap_method(public_id)
+    on delete cascade
+    on update cascade,
+  from_attribute text not null
+    constraint from_attribute_must_not_be_empty
+       check(length(trim(from_attribute)) > 0) 
+    constraint from_attribute_must_be_less_than_1024_chars
+      check(length(trim(from_attribute)) < 1024),
+  to_attribute text not null 
+    constraint to_attribute_valid_values 
+      check (lower(to_attribute) in ('fullname', 'email')), -- intentionally case-sensitive matching
```

Comment:
Should we define a separate enum table for this like we do elsewhere instead? At what point does this warrant an enum table?

https://github.com/hashicorp/boundary/blob/d5fed1091826ff01778c106dc054f0a2a66f1e88/internal/db/schema/migrations/oss/postgres/11/01_server_type_enum.up.sql#L3-L19

Reply:
I don't believe this warrants a separate table.